src/ContentTest.java

```diff
@@ -0,0 +1,39 @@
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+public class ParserTest {
+    private String WithUnicode = "ЙWithUnicode";
+    private String WithoutUnicode = "WithoutUnicode";
+    private File file;
+
+    @Before
+    public void init () {
+        file = new File("tests/contentTest.txt");
+    }
+    @Test
+   public void unicodeContent() throws IOException {
+        FileDataSource contentForWorkingWithFile = new FileDataSource(file);
+                contentForWorkingWithFile.saveContent(WithUnicode);
+                String actual = contentForWorkingWithFile.getContent();
+                Assert.assertEquals("ЙWithUnicode",actual);
+    }
+    @Test
+    public void nonUnicodeContent() throws IOException{
+        Content content = new NoUnicode(new FileDataSource(file));
+        content.saveContent(WithoutUnicode);
+        String read = content.getContent();
+        Assert.assertEquals("WithoutUnicode", read);
+    }
+
+
+
+    @After
+     public void after(){
```

Comment:
@dwaynewade0033 https://docs.google.com/document/d/1u2codBL2PL9j9jpl-SurrClnhUG35RPMWV0FEJa3LU0/edit#heading=h.c8uik21w07ew

Reply:
@dwaynewade0033 but for this particular case, I'd suggest to have some predefined resources, like `unicode.txt` and `non_unicode.txt` and use it within the test